src/course-outline/section-card/SectionCard.jsx

```diff
@@ -30,13 +38,19 @@ const SectionCard = ({ section, children, onOpenHighlightsModal }) => {
     setIsExpanded((prevState) => !prevState);
   };
 
+  const handleMenuButtonClick = () => {
```

Comment:
please change to `handleClickMenuButton`

Reply:
Done